setup.py

```diff
@@ -138,14 +138,30 @@ def initialize_options(self) -> None:
 ]
 
 db_dependencies = [
-    # sqlalchemy 1.3.16 seems to have (accidentally?) introduced
-    # a breaking change that affects sqlalchemy-redshift:
+    # Without this, pip 2.20.4 with "--use-feature=2020-resolver" will
```

Comment:
This file, one of the 3 largest in the project, increased in size to 373 lines.  The total size of those files is now 3415 lines (target: 1087).  Is this file complex enough to refactor?